src/main/java/uk/gov/ida/verifyserviceprovider/configuration/ConfigurationConstants.java

```diff
@@ -1,8 +1,14 @@
 package uk.gov.ida.verifyserviceprovider.configuration;
 
 public interface ConfigurationConstants {
-    String PRODUCTION_VERIFY_TRUSTSTORE_NAME = "verify-production-truststore.ts";
-    String TEST_VERIFY_TRUSTSTORE_NAME = "verify-test-truststore.ts";
+    String DEFAULT_TRUST_STORE_PASSWORD = "bj76LWZ+F5L1Biq4EZB+Ta7MUY4EQMgmZmqAHh";
```

Comment:
Please add the following comment before this line to let developers know that trust stores do not have private keys and the password should not be used for anything sensitive.
```
     /**
     * Note: our trust stores do not contain private keys,
     * so this password does not need to be managed securely.
     *
     * This password MUST NOT be used for anything sensitive, since it is open source.
     */
```